sdk/apimanagement/api-management-custom-widgets-scaffolder/rollup.config.bin.mjs

```diff
@@ -2,7 +2,7 @@ import MagicString from "magic-string";
 import fs from "fs";
 import nodeResolve from "@rollup/plugin-node-resolve";
 import path from "path";
-import pkg from "./package.json";
+import pkg from "./package.json" assert { type: "json" };
```

Comment:
Note JSON import assertions backported: v16.14.0, however, to use them for versions less than v17.5.0 you need to use this flag `--experimental-json-modules`

Reply:
Went with a simple `readFile` for now in https://github.com/Azure/azure-sdk-for-js/pull/27377/commits/a74fe46ad58ddea4173c3c7261bdc0652a7bfba3